### DISCUSSION TITLE
Backticks shouldn't wrap independently of text

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#142548] [Mobile] [answered]
GitHub Mobile beta for Android 

1. Open https://github.com/argoproj/argo-cd/pull/20097 to the files view
2. Scroll to 
![Screenshot_20241024-192536.png](https://github.com/user-attachments/assets/4fbe96c6-c1c7-49f3-b96e-a907f5dea8fc)

## Actual Results
See both a bare backtick at the beginning and the end of a line

## Expected Results
Backticks should wrap with adjacent words

**Comment by eliperkins** [comment, marked as answer]
Hey @jsoref, thanks for this feedback! This is a great catch, there definitely shouldn't be a line-break immediately following a backtick. This seems like a bug to me and not intentional behavior, so I've filed an issue to track fixing this. I'll follow up with you when the fix has been deployed!

Thanks for helping improve the quality of GitHub Mobile 🙇

**Comment by MasterInQuestion** [comment]
The quirkiness is somewhat similar:
    https://github.com/orgs/community/discussions/139005#discussioncomment-11046170
    .
    Though for your case it's mostly of different cause:
    Supposedly ["white-space: nowrap"](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space) alike with peculiar ["word-break"](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break), ["overflow-wrap"](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap) etc.
